frontend/src/components/auth/Logout.tsx

```diff
@@ -32,7 +32,8 @@ const Logout = () => {
   useEffect(() => {
     if (error) {
       // TODO: Implement Error State
-      console.log(error);
+      // eslint-disable-next-line no-alert
```

Comment:
linting errors

Reply:
Does your linter give you a blocking error when you don't include this?

Reply:
the linter blocked me due to some weird local changes to prettier. I know this isn't technically blocking on master, but doesnt hurt to include it - `alerts` and `console.logs` are generally bad practice